src/contract/basic_node_signer.ts

```diff
@@ -16,14 +17,25 @@ import type { Client } from "./client";
 export const basicNodeSigner = (
   keypair: Keypair,
   networkPassphrase: string,
-) => ({
+): {
+  signTransaction: KitActions['signTransaction'];
+  signAuthEntry: KitActions['signAuthEntry'];
```

Comment:
Is this whole import worth it just for these few lines?

Reply:
I do think it would be better to use "official" types than to copy-paste them, mostly to guarantee compatibility with a certain standard, but agree that copy-pasting them for now seems like the best choice.